### PR TITLE
[release-danube] Changelog v1.10.1

### PR DESCRIPTION
**Pull request by karimnaaji**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
+## 1.10.1
+
+### 🐞 Bug fixes
+* Fix multi-finger gestures that touch markers ([#9675](https://github.com/mapbox/mapbox-gl-js/issues/9675), fixed by [#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
+* Fix bug where `map.isMoving()` returned true while map was not moving ([#9647](https://github.com/mapbox/mapbox-gl-js/issues/9647), fixed by [#9679](https://github.com/mapbox/mapbox-gl-js/pull/9679))
+* Fix regression that prevented touchmove events from firing during gestures ([#9676](https://github.com/mapbox/mapbox-gl-js/issues/9676), fixed by [#9685](https://github.com/mapbox/mapbox-gl-js/pull/9685))
+* Fix `image` expression evaluation which was broken under certain conditions ([#9630](https://github.com/mapbox/mapbox-gl-js/issues/9630), fixed by [#9685](https://github.com/mapbox/mapbox-gl-js/pull/9668))
+* Fix bug when using a combined expression of `within` and other expressions as a filter ([#9605](https://github.com/mapbox/mapbox-gl-js/issues/9605), fixed by [#9611](https://github.com/mapbox/mapbox-gl-js/pull/9611))
```
Comment (arindam1993) [inline CHANGELOG.md:8]:
```suggestion
* Fix nested `within` expressions in filters not evaluating correctly.  ([#9605](https://github.com/mapbox/mapbox-gl-js/issues/9605), fixed by [#9611](https://github.com/mapbox/mapbox-gl-js/pull/9611))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
+## 1.10.1
+
+### 🐞 Bug fixes
+* Fix multi-finger gestures that touch markers ([#9675](https://github.com/mapbox/mapbox-gl-js/issues/9675), fixed by [#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
```
Comment (arindam1993) [inline CHANGELOG.md:4]:
```suggestion
* Fix markers interrupting touch gestures. ([#9675](https://github.com/mapbox/mapbox-gl-js/issues/9675), fixed by [#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,21 @@
+## 1.10.1
+
+### 🐞 Bug fixes
+* Fix markers interrupting touch gestures ([#9675](https://github.com/mapbox/mapbox-gl-js/issues/9675), fixed by [#9683](https://github.com/mapbox/mapbox-gl-js/pull/9683))
+* Fix bug where `map.isMoving()` returned true while map was not moving ([#9647](https://github.com/mapbox/mapbox-gl-js/issues/9647), fixed by [#9679](https://github.com/mapbox/mapbox-gl-js/pull/9679))
+* Fix regression that prevented touchmove events from firing during gestures ([#9676](https://github.com/mapbox/mapbox-gl-js/issues/9676), fixed by [#9685](https://github.com/mapbox/mapbox-gl-js/pull/9685))
```
Comment (asheemmamoowala) [inline CHANGELOG.md:6]:
```suggestion
* Fix regression that prevented `touchmove` events from firing during gestures ([#9676](https://github.com/mapbox/mapbox-gl-js/issues/9676), fixed by [#9685](https://github.com/mapbox/mapbox-gl-js/pull/9685))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+* Fix `image` expression evaluation which was broken under certain conditions ([#9630](https://github.com/mapbox/mapbox-gl-js/issues/9630), fixed by [#9685](https://github.com/mapbox/mapbox-gl-js/pull/9668))
+* Fix nested `within` expressions in filters not evaluating correctly  ([#9605](https://github.com/mapbox/mapbox-gl-js/issues/9605), fixed by [#9611](https://github.com/mapbox/mapbox-gl-js/pull/9611))
+* Fix potential undefined paint variable in StyleLayer ([#9688](https://github.com/mapbox/mapbox-gl-js/pull/9688)) (h/t [mannnick24](https://github.com/mannnick24))
```
Comment (asheemmamoowala) [inline CHANGELOG.md:9]:
```suggestion
* Fix potential `undefined` paint variable in `StyleLayer` ([#9688](https://github.com/mapbox/mapbox-gl-js/pull/9688)) (h/t [mannnick24](https://github.com/mannnick24))
```

File: CHANGELOG.md (hunk 3 above)
Comment (karimnaaji) [reply CHANGELOG.md:6]: Addressed in https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.10.1 and https://github.com/mapbox/mapbox-gl-js/pull/9697/commits/e3b97d9b6ce142cd4b6e927f1a317d94066a8a91

File: CHANGELOG.md (hunk 4 above)
Comment (karimnaaji) [reply CHANGELOG.md:9]: Addressed in https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.10.1 and https://github.com/mapbox/mapbox-gl-js/pull/9697/commits/e3b97d9b6ce142cd4b6e927f1a317d94066a8a91

**Pull request merged**